contracts/rmrk/multiresourcenft/manifest.json

```diff
@@ -0,0 +1,1114 @@
+        {
+          "name": "Max supplly",
+          "description": "Returns a number of max tokens in the collection",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "mint(address,uint256)",
+      "name": "Mint",
+      "description": "Mints a new token to an address",
+      "parameterDecorators": [
+        {
+          "name": "To address",
+          "description": "Minted token owner",
+          "recommendedTypes": []
+        },
+        {
+          "name": "Number of tokens to mint",
+          "description": "Quantity of NFTs to mint",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "name()",
+      "name": "Name",
+      "description": "Collection name",
+      "parameterDecorators": [],
+      "returnDecorators": [
+        {
+          "name": "Name",
+          "description": "Returns name of the collection",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "owner()",
+      "name": "Owner",
+      "description": "Collection owner",
+      "parameterDecorators": [],
+      "returnDecorators": [
+        {
+          "name": "Owner",
+          "description": "Returns address of owner of the collection",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "ownerOf(uint256)",
+      "name": "Owner Of",
+      "description": "Returns token owner for requested token id",
+      "parameterDecorators": [
+        {
+          "name": "Token ID",
+          "description": "Token ID",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [
+        {
+          "name": "Owner",
+          "description": "Returns address of owner of the token",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "pricePerMint()",
+      "name": "Price per mint",
+      "description": "Price for minting of a new token",
+      "parameterDecorators": [],
+      "returnDecorators": [
+        {
+          "name": "Price",
+          "description": "Returns price of the minting of a new token",
+          "recommendedTypes": []
+        }
+      ],
+      "emittableEvents": []
+    },
+    {
+      "signature": "rejectAllResources(uint256)",
+      "name": "Reject all resources",
+      "description": "Rejects all resources for a given token id",
+      "parameterDecorators": [
+        {
+          "name": "Token ID",
+          "description": "Token ID",
+          "recommendedTypes": []
+        }
+      ],
+      "returnDecorators": [],
+      "emittableEvents": []
+    },
+    {
+      "signature": "rejectResource(uint256,uint256)",
+      "name": "Reject a resource",
+      "description": "Rejects a resource for a given token id and resource id",
```

Comment:
Rejects a pending resource for a given token id and index (of pending resources array)